.github/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -18,10 +18,22 @@ If so, please describe the problem
 
 Include any specific requirements you need
 
+**Notify relevant community channels**
+
+Consider raising this topic on [Slack] in the #development channel or during a
```

Comment:
Might want to give some instructions if the sig is not obvious.

Reply:
Do you have an idea what that might look like?

Ideas:
- Include any area you think might be relevant, the recipients can decide which area it's more related to
- Use the CODEOWNERS file to find which files you might change and locate teams that way

Reply:
I think going broad and then having the sigs understand that they need unmark themselves would be a good start. Pinging on development as well is ok -- but I've found the response rate might not be predictable?

Reply:
I think "response rate unpredictability" is just how it is in open source - as a group of volunteers, nobody is under any obligation to respond. But the broad notification to the development channel seems like it's the best place to find other folks who are interested, and the wider the interest in an idea, the more likely that SIG folks and committers will want to help

Reply:
I accepted Bill's wording and also amended to simplify the language:

> If you don't know which team to include or you haven't gotten feedback, consider raising this topic on [Slack] in the #development channel or during a [community meeting] for awareness.